doc/getting_started/setup_dv.md

```diff
@@ -111,8 +111,15 @@ $ git checkout -b <temp_branch> FETCH_HEAD
 ```console
 $ util/dvsim/dvsim.py hw/top_earlgrey/dv/top_earlgrey_sim_cfgs.hjson -i smoke --fixed-seed=1
 ```
-We can only run the failed test with `--fixed-seed=1` to reproduce the failure.
-Assume there is a failure in the `uart_smoke` test. Here is the run command to reproduce it.
+Since the CI runs tests with pseudo-random behaviour driven from 'seed' numbers, to be confident of reproducing the failure we must supply the exact seed that CI used.
+
+Assume there is a failure in the `uart_smoke` test. To reproduce this with the DV simulation environment we use the following command, remembering to replace '<seed>' with the seed number, and to choose an appropriate waveform '<format>':
+
+
```

Comment:
Did you mean to put an extra newline here? If not we should stick to one for consistency.